independent-publisher-connectors/rSpaceX/settings.json

```diff
@@ -0,0 +1,9 @@
+{

```

Comment:
@troystaylor Please remove this file and add readme.md.

Reply:
Fixed